### ISSUE TITLE
Support for reverse proxy

### ISSUE THREAD

**Issue by quinz** [geek1011/BookBrowser#55]
Hi there and thanks for a promising looking project!

I have tried to run BookBrowser on my home server behind a reverse proxy (nginx), but using an additional url block (such as /bookbrowser) doesn't seem to work. It breaks the links of the app.

e.g. mydomain.lan/bookbrowser/ redirects me to mydomain.lan/books which causes 404. The app should understand to use mydomain.lan/bookbrowser/books instead

Could you consider adding a support for an optional prefix for the URI paths?

**Comment by geek1011** [comment]
Yep. I'll do that when I finish the next version (a complete rewrite). I'll be away for the next few weeks though, so it may be a while.

**Comment by quinz** [comment]
Thanks man, enjoy your time off.

**Comment by sKanoodle** [comment]
@quinz A workaround for your problem would be to use a subdomain like for example bookbrowser.mydomain.lan, this would eliminate the need to rewrite the url.
This could complicate name resolution if you don't use a DNS server for your LAN domain lookup, though.

**Comment by quinz** [comment]
Hi @sKanoodle, unfortunately I don't have the luxury of subdomains as my internet facing address is from dyndns. My example urls in the first post are confusing, sorry for that. Your tip can be useful for someting else though!

**Comment by quinz** [comment]
Hello, is there any estimate when the new release with this feature would be out?

**Comment by geek1011** [comment]
Sorry, it's been delayed quite a bit, as I have quite a few other things which have a higher priority (BookBrowser works for me as-is right now). I'll be doing it when I get around to it. I don't have any estimate right now, other than sometime this year.
